index.html

```diff
@@ -1,67 +1,116 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-	<title>Derek Graham</title>
-	    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Derek Graham</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="vendor/styles/normalize.css">
     <link rel="stylesheet" href="styles/icons.css">
     <link rel="stylesheet" href="styles/base.css">
     <link rel="stylesheet" href="styles/layout.css">
-    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/modules/nav.css">
+    <link rel="stylesheet" href="styles/modules/about.css">
+    <link rel="stylesheet" href="styles/modules/projects.css">
+    <script id='project-template' type="text/x-handlebars-template">
+        <article data-category="{{category}}">
+            <header>
+                <h5>{{name}}</h5>
+                <div class="byline">
+                    By
+                    <address><a href="">{{maker}}</a></address>
+                    created
+                    <time title="{{publishedDate}}" pubdate="{{publishedDate}}"" datetime="2000-01-01">{{relativePubDate}}</time>
+                </div>
+            </header>
+            <section class="project-body">{{description}}</section>
+            {{#if media.imageURL}}
+            <section class="project-image">
+                <a class='project-image-link' href=""><img class='project-image' src="{{media.imageURL}}"></a>
+            </section>
+            {{/if}} {{#if media.video}}
```

Comment:
You can do else in Handlebars, unless you want to have an image and a video at the same time.